auction9-backend/handler.js

```diff
@@ -53,3 +53,24 @@ export const getActiveAuctions = async (event, context) => {
     };
   }
 };
+
+/* postAuction - creates new auction
+ * POST: /newauction
+ */
+export const postAuction = async (event, context) => {
+	try {
+		let reqBody = JSON.parse(event.body);
+		let sqlInsert = await mysql.query('INSERT INTO tbl_auction (`title`, `description`, `date_from`, `date_to`, `price`, `status`, `created_by`, `winner`) VALUES (?, ?, ?, ?, ?, ?, ?, ?)',
```

Comment:
Once we implement authentication we'll need to change this so it stores `created_by` based on session info. 
Another thing: does it makes sense to set the winner in the moment of creation?